libraries/streaming-e2e-tests/react-app/public/index.html

```diff
@@ -9,6 +9,15 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <meta http-equiv="Content-Security-Policy"
```

Comment:
Perhaps add a comment with links to issues? Either here or in the readme?

We don't want people in the future to come back and remove the "Content-Security-Policy" because it's blocking them as individuals.

Reply:
Done!

Updated README and added comments to `App.js` and `index.html` w/links to bb-js issue, to provide reasoning behind the CSP addition